### ISSUE TITLE
Html Report is not getting generated when there are 0 warnings and errors

### ISSUE THREAD

**Issue by chandru9279** [ivan-vesely/gulp-jshint-html-reporter#2]
As the title says.

When I introduce an error, I can see the report being generated properly.

I have configured a custom tab in our CI pipeline to show the HTML report (which is part of the artifacts of our build), and would like to see a "green no errors" report there, as opposed to nothing.

**Comment by ivan-vesely** [comment]
@chandru9279 It is supposed to do nothing when there is no error.
You think it should generate html report in case there is no jshint error?

**Comment by chandru9279** [comment]
Would be great if it generated an "All Okay no errors" report file. I am using [Custom Tabs](http://www.thoughtworks.com/products/docs/go/12.2/help/managing_artifacts_and_reports.html) on my CI, and would be great if it reported no JSHINT issues, and a list of files that were processed.

**Comment by robertdunaway** [comment]
I have to agree with chandru9279.

I just starting using this reporter today and I must say I love it.  Great job.

I would like for it to still create an html page if there are no error found.  The reason for this is after I correct a problem the html page still reflects the JavaScript error.

For now my work-a-round is to leave at least one error in my JavaScript so this report is always generated and I can see when my corrections take effect.

Great tool.  Would be very helpful if a simple html file was still generated even if not errors are found.

Thanks,
Bob

**Comment by ivan-vesely** [comment]
Yeah, that's fair enough. Maybe can i expect PR from other open source contributors?

**Comment by TjRus** [comment]
Why not just use `gulp-clean` package before jshint to clean the previous report?
In thin case if there is no report file - everything is ok with your code.
